### DISCUSSION TITLE
Mi pokedex

### DISCUSSION THREAD

**Discussion by pavg92** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#549] [Show and tell]
Que tal!

Comparto mi pokedex, tardé en realizarla debido a que no había tenido tiempo por algunos asuntos que tenia.
 
Acepto cualquier comentario para poder mejorarla

* [Repositorio](https://github.com/pavg92/front-end-semana4)
* [Despliegue](https://pavg92.github.io/front-end-semana4/)

![screencapture-pavg92-github-io-front-end-semana4-2022-03-28-00_47_12](https://user-images.githubusercontent.com/56892136/160341497-2b7c776a-7f50-48bd-9300-79e6b8bbb49c.png)

**Comment by doguedogue** [comment]
Está genial

Prueba con estas imagenes si gustas, tienen mejor resolución
`data.sprites.other.home.front_default;`
